<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Main">
         <attribute name="title">
          <string>状态检测</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,4">
            <property name="leftMargin">
             <number>7</number>
            </property>
            <property name="topMargin">
             <number>7</number>
            </property>
            <property name="rightMargin">
             <number>7</number>
            </property>
            <property name="bottomMargin">
             <number>7</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
              <property name="spacing">
               <number>1</number>
              </property>
              <property name="leftMargin">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSlider" name="Slider_speed">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>-100</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7" stretch="9,1">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <widget class="QCustomPlot" name="Plot" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2,2,3">
                <property name="spacing">
                 <number>20</number>
                </property>
                <property name="leftMargin">
                 <number>7</number>
                </property>
                <property name="topMargin">
                 <number>7</number>
                </property>
                <property name="rightMargin">
                 <number>7</number>
                </property>
                <property name="bottomMargin">
                 <number>7</number>
                </property>
                <item>
                 <widget class="QPushButton" name="Button_buz">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Buz</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Button_clear">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>清屏</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="Checkbox_autoscale">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>自动缩放</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Button_zero">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Courier New</family>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="9,1">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>设置</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0" colspan="2">
                  <widget class="QPushButton" name="Button_Link">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>连接</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QTextBrowser" name="TextBrowser"/>
                 </item>
                 <item row="2" column="0" colspan="2">
                  <widget class="QPushButton" name="Button_timesync">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>同步时间</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="Button_sdstart">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>SD记录</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QPushButton" name="Button_sdstop">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>SD停止</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QPushButton" name="Button_lighton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Light On</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QPushButton" name="Button_lightoff">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Light Off</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QCheckBox" name="Check_control">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>AcadEref</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>获取控制权</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QRadioButton" name="Radio_Speed">
                   <property name="text">
                    <string>绘制速度曲线</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QRadioButton" name="Radio_Directon">
                   <property name="text">
                    <string>绘制方向曲线</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0">
                  <widget class="QPushButton" name="Button_start">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>启动</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="1">
                  <widget class="QPushButton" name="Button_stop">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>停止</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="Slider_direction">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>-100</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Setting">
         <attribute name="title">
          <string>设置</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>231</width>
            <height>91</height>
           </rect>
          </property>
          <property name="title">
           <string>连接设置</string>
          </property>
          <widget class="QComboBox" name="Edit_IP">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>60</width>
             <height>16</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>服务器IP：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>60</width>
             <height>16</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>目标端口：</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="Edit_Port">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>60</y>
             <width>111</width>
             <height>18</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>3721</number>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="Button_restore">
          <property name="geometry">
           <rect>
            <x>520</x>
            <y>500</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>重设</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Button_apply">
          <property name="geometry">
           <rect>
            <x>650</x>
            <y>500</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>应用</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Help">
         <attribute name="title">
          <string>说明</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
